observer/obs_conf.go

```diff
@@ -0,0 +1,58 @@
+package observer
+
+import (
+	"errors"
+
+	"github.com/letsencrypt/boulder/cmd"
+	"github.com/letsencrypt/boulder/observer/plugins"
+)
+
+var (
+	errNewObsNoMons  = errors.New("observer config is invalid, 0 monitors configured")
+	errNewObsEmpty   = errors.New("observer config is empty")
+	errNewObsInvalid = errors.New("observer config is invalid")
+)
+
+// ObsConf is exported to receive the supplied observer config
+type ObsConf struct {
+	Syslog    cmd.SyslogConfig `yaml:"syslog"`
+	DebugAddr string           `yaml:"debugAddr"`
+	Modules   []plugins.Conf   `yaml:"modules"`
+	Timeout   int              `yaml:"timeout"`
+	NewMons   []MonConf        `yaml:"monitors"`
+}
+
+func (n *ObsConf) validateMonConfs() error {
+	i := 0
+	for _, m := range n.NewMons {
+		if !m.Enabled {
+			continue
+		}
```

Comment:
If we have an Enabled bit, I would continue validating even  the `!Enabled` configs.